src/transformers/models/sam/modeling_sam.py

```diff
@@ -743,7 +759,7 @@ def get_rel_pos(self, q_size: int, k_size: int, rel_pos: torch.Tensor) -> torch.
         """
         max_rel_dist = int(2 * max(q_size, k_size) - 1)
         # Interpolate rel pos if needed.
-        if rel_pos.shape[0] != max_rel_dist:
+        if True: # rel_pos.shape[0] != max_rel_dist:
```

Comment:
If we fall into this at tracing-time, will it alway be the case?

Basically my question is if it's input shape dependent?